packages/amplify-e2e-tests/src/__tests__/geo_1.test.ts

```diff
@@ -0,0 +1,56 @@
+import { 
```

Comment:
can we name the test files as `geo-add.test.ts` or equivalent to make it more clear?

Reply:
Sure. The previous naming was just following the CLI's convention.